features/step_definitions/machine_set.rb

```diff
@@ -0,0 +1,9 @@
+# machineset supporting steps
+#
+Given /^I store all machinesets to the#{OPT_SYM} clipboard$/ do |cb_name|
+  cb_name ||= :machinesets
+  @result = admin.cli_exec(:get, resource: 'machineset', n: 'openshift-machine-api', o: 'yaml')
+  cb[cb_name] = @result[:parsed]["items"].map { |i| i['metadata']['name']}
```

Comment:
Can we store machineset objects here instead of string names? `MachineSet.list(...)`?

Reply:
Sure, but that would mean we have to save project context and restore it back